### PR TITLE
MIMIR-1280: Removed unused properties for highchart source 

### PR DESCRIPTION
**Pull request by annesiri**

### CODE REVIEW

**Comment by annesiri** [comment]
NB! Må ikke merges! Forretning oppdaget at det fortsatt ligger kilder i disse gamle kildefeltene.

File: src/main/resources/site/parts/highchart/Highchart.jsx [hunk 1]
```diff
@@ -102,18 +103,29 @@ function Highchart(props) {
...
+  function renderHighchartsFooter(highchart) {
+    return (
+      <Col>
+        {highchart.footnoteText ? (
+          <Row className='footnote mb-4 mb-md-5'>
+            <Col>{highchart.footnoteText}</Col>
           </Row>
         ) : null}
+        {highchart.creditsEnabled
+          ? ensureArray(highchart.sourceList).map((source) => renderHighchartsSource(source))
```
Comment (ssb-cgn) [inline src/main/resources/site/parts/highchart/Highchart.jsx:127]: SourceList is already a array, dont thing you need ensureArray

File: src/main/resources/site/parts/highchart/Highchart.jsx [hunk 2]
```diff
@@ -102,18 +103,29 @@ function Highchart(props) {
...
     return (
-      <Col className='col-12'>
-        {highchart.footnoteText ? <Row className='col-12 footnote'>{highchart.footnoteText}</Row> : null}
-        {highchart.creditsEnabled ? (
-          <Row className='mt-4 mt-md-5 highcharts-source'>
-            <Col className='col-12 fw-bold mb-0'>{props.phrases.source}:</Col>
-            <Col className='col-12'>
-              <Link href={highchart.creditsHref}>{highchart.creditsText}</Link>
-            </Col>
+      <Row>
```
Comment (ssb-cgn) [inline src/main/resources/site/parts/highchart/Highchart.jsx:108]: Add key to avoid warning in vs code

**Review by ssb-cgn — CHANGES_REQUESTED** [review]
Added some comments

File: src/main/resources/site/parts/highchart/Highchart.jsx (hunk 2 above)
Comment (annesiri) [reply src/main/resources/site/parts/highchart/Highchart.jsx:108]: I don't get any warnings in VS code. Guess there is some config or extension I am missing, can you show me?

**Comment by annesiri** [comment]
Fixed comments. Ready for QA once more.

**Review by ssb-cgn — APPROVED** [review]
Nice 👍

**Pull request merged**